sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample5_ImportProject.md

```diff
@@ -120,11 +120,11 @@ var trainData = new
 Console.WriteLine($"Training project {projectName}...");
 Operation<BinaryData> trainOperation = client.Train(WaitUntil.Completed, projectName, RequestContent.Create(trainData));
 
-    // Deploy the model.
-    var deployData = new
-    {
-        trainedModelLabel = "Sample5",
-    };
+// Deploy the model.
+var deployData = new
+{
+    trainedModelLabel = "Sample5",
```

Comment:
If we wanted to illustrate `JsonPropertyNames.CamelCase` usage in RequestContent.Create(), we could consider doing it here?

Reply:
Will be a separate PR.